test_pytest.py

```diff
@@ -309,3 +309,190 @@ def test_edu_update():
                                   json=updated_example_education)
     response = app.test_client().get('/resume/education')
     assert response.json[item_id] == updated_example_education
+
```

Comment:
Well done!
Great tests!

It's recommendable to test edge cases. Please create a test for when the user doesn't send the corrections parameter.

Reply:
The last test covers for no corrections parameter passed. Check out `test_spell_check_no_corrections()`